app/controllers/candidate_interface/application_choices_controller.rb

```diff
@@ -0,0 +1,65 @@
+module CandidateInterface
+  class ApplicationChoicesController < CandidateInterfaceController
+    before_action :redirect_to_dashboard_if_submitted
+
+    def index
+      @course_choices = current_candidate.current_application.application_choices
```

Comment:
We use `course_choice` or similar in a few places to refer to an application choice. Would it be less confusing to just call the `application_choice` or `application_choices` etc.?